src/config/db.ts

```diff
@@ -19,4 +19,8 @@ const db = new Sequelize({
   },
 });
 
+db.sync({ alter: true }).then(() =>
```

Comment:
This is used to sync all the data tables instead of only one in Todo. Very nice!